rmf_battery/include/rmf_battery/agv/MechanicalSystem.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef RMF_BATTERY__AGV__MECHANICALSYSTEM_HPP
+#define RMF_BATTERY__AGV__MECHANICALSYSTEM_HPP
+
+#include <rmf_utils/impl_ptr.hpp>
+
+namespace rmf_battery {
+namespace agv {
+
+
+class MechanicalSystem
+{
+public:
+
+  MechanicalSystem(
+    double mass,
+    double inertia,
```

Comment:
I assume this is meant to be the moment of inertia about the z-axis. That should be made more explicit. Simply saying "inertia" usually refers to mass.

Reply:
Yes this is the moment of inertia about the Z-Axis. Will update the documentation

Reply:
https://github.com/osrf/rmf_core/pull/217/commits/d416f83a321ccf04a08fd1711248194babee5fbc